internal/resources/grafana/resource_data_source_permission.go

```diff
@@ -28,8 +27,9 @@ func ResourceDatasourcePermission() *schema.Resource {
 		DeleteContext: DeleteDatasourcePermissions,
 
 		Schema: map[string]*schema.Schema{
+			"org_id": orgIDAttribute(),
 			"datasource_id": {
-				Type:        schema.TypeInt,
+				Type:        schema.TypeString,
```

Comment:
Why changing this to a string, when it seems all of the uses below treat it as an int?